frontend/src/views/producer/dashboard/edition-form/Editor.jsx

```diff
@@ -146,6 +127,19 @@ const Editor = observer(
               value={this.eventDraft.location}
             />
           </Row>
+          <Row
+            fullsize
+            style={{
+              backgroundColor: 'rgba(0,0,0, .1)',
```

Comment:
Also here you have quite a lot of styles inline.. how about just adding a className and putting the styles in a css file?